src/index.tsx

```diff
@@ -28,16 +28,15 @@ function Site() {
     <Router>
       <Navbar items={SiteData.NavbarItems} />
       <Routes>
-        <Route exact path={SiteData.HomeTarget} element={<Home />} />
-        <Route exact path={SiteData.AboutUsTarget} element={<About />} />
-        <Route exact path={SiteData.EventsTarget} element={<Events />} />
-        <Route exact path={SiteData.GuestsTarget} element={<Hosts />} />
-        <Route exact path={SiteData.PiaTarget} element={<PassionInAction />} />
-        <Route exact path={SiteData.NetworkTarget} element={<Network />} />
-        <Route exact path={SiteData.JoinTarget} element={<Join />} />
-        <Route exact path={SiteData.OpenDayTarget} element={<OpenDay />} />
+        <Route path={SiteData.HomeTarget} element={<Home />} />
+        <Route path={SiteData.AboutUsTarget} element={<About />} />
+        <Route path={SiteData.EventsTarget} element={<Events />} />
+        <Route path={SiteData.GuestsTarget} element={<Hosts />} />
+        <Route path={SiteData.PiaTarget} element={<PassionInAction />} />
+        <Route path={SiteData.NetworkTarget} element={<Network />} />
+        <Route path={SiteData.JoinTarget} element={<Join />} />
+        <Route path={SiteData.OpenDayTarget} element={<OpenDay />} />
```

Comment:
Are you sure we dont need to use `exact` here?

Reply:
It looks to be working without it so maybe it's fine

Reply:
Just FYI, from react-router v6 i'ts not necessary to use "exact" because it's the default.
https://stackoverflow.com/questions/69866581/property-exact-does-not-exist-on-type

Reply:
I see, good catch